waku/node/v2/wakunode2.nim

```diff
@@ -170,8 +170,11 @@ proc start*(node: WakuNode, conf: WakuNodeConf) {.async.} =
   node.libp2pTransportLoops = await node.switch.start()
 
   # NOTE WakuRelay is being instantiated as part of creating switch with PubSub field set
-  #
-  # TODO Mount Waku Store and Waku Filter here
+  let storeProto = WakuStore.init()
+  node.switch.mount(storeProto)
+
+  let wakuRelayProto = cast[WakuRelay](node.switch.pubSub.get())
+  wakuRelayProto.addFilter("store", storeProto.filter())
```

Comment:
```suggestion
  wakuRelay.addFilter("store", storeProto.filter())
```

Reply:
I don't understand what "store" as a string is here tbh

Reply:
The way it is set up now it should be a unique identifier, so that the filter can be removed afterwards.
But it could also be a topic + handler combo.

Reply:
@kdeme I could just set it to the ProtoCodec